pkgs/top-level/all-packages.nix

```diff
@@ -20208,6 +20208,8 @@ in
   inherit (gnome3) evince;
   evolution-data-server = gnome3.evolution-data-server;
   evolution-ews = callPackage ../applications/networking/mailreaders/evolution/evolution-ews { };
+  evolution = callPackage ../applications/networking/mailreaders/evolution/evolution { };
+  evolutionBundle = callPackage ../applications/networking/mailreaders/evolution/evolution/wrapper.nix { };
```

Comment:
Also since this is a function rather than derivation, it cannot be used directly. It would be nice if this was a derivation with a default list of plug-ins (either all or none) that could be changed by `.override` just like the other plug-in wrappers allow.

Reply:
done